notes-webapp/src/main/webapp/skin/less/notes/notes.less

```diff
@@ -7,7 +7,8 @@
 @import "Notifications/Style.less";
 @import "Sprites/Style.less";
 @import "UIWikiRichTextEditor/Stylesheet.less";
-
+@import "../../../ckeditor/skins/common/extendedRichContent.less";
+@import (less) "../../../ckeditor/plugins/codesnippet/lib/highlight/styles/monokai_sublime.css";
```

Comment:
Not sure that this is the best place to define the reusable Less files, since we have platform-ui for it. I will change it anyway when starting to work on SNVs V2. Nothing to do more from your part, I 'll take it from here, but just to share the best practice for future devs.